### ISSUE TITLE
donation list flags

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#188]
I've recently needed to use the `donation list` command and got a bit stuck without understanding how it work, because its flags are not perfectly aligned with the same essential flags of the other command.

Specifically I'd propose to make the following changes:
1) What if we rename the `--all-states` flag into `--full-list[-of-states]` in order to free the `-a` flag and to use it as the `--address` flag (instead of `-e`) ?
2) If the previous point is acceptable maybe we can also allow using the `-m` flag as alternative to the `-a` flag, instead of using them together. Or maybe we may use the `-a` flag with the keyword `my` or `mine` afterwards, instead of the `-m` flag?

**Comment by d5000** [comment]
I have looked into it and the problem with your first proposal is that I had aligned this `-a` flag in a way it matches the behavior of `proposal list`, where also `-a` shows the list of _all_ proposals. It cannot be changed to `-f` in the proposal list because this is occupied by `--find`, and this is also a keyword I use a lot in other list commands.

Of corse the `--examine_address` flag is not ideal.  The ideal flag would actually be `--donor_address` but I don't want to sacrifice `-d` for `--debug`.

If you have proposals for alternatives which don't include a renaming of the `-a` flag I'm open to this. Anyway this is not much of a priority now, can also stay open to be thought about until after the PoB token launch.
